src/app/components/SearchContainer/SearchContainer.jsx

```diff
@@ -23,36 +26,45 @@ class SearchContainer extends React.Component {
     this.boundActions = bindActionCreators(searchActions, dispatch);
   }
 
+  componentDidUpdate() {
+    ReactDOM.findDOMNode(this).scrollIntoView();
+  }
+
   render() {
-    const { query } = (this.props.location) ? this.props.location : '';
-    const userQuery = (query && query.q) ? query.q : this.props.searchQuery;
+    const { query } = this.props.location;
+    const searchQuery = (query && query.q) ? query.q : this.props.searchQuery;
     const selectedField = (query && query.field) ? query.field : this.props.searchField;
+    const pageType = (_isEmpty(this.props.searchResults)) ? 'home' : 'results';
+
     return (
       <main id="mainContent">
-        <div className="nypl-page-header">
-          <nav aria-label="Breadcrumbs" className="nypl-breadcrumbs" />
-        </div>
-        <div className="nypl-full-width-wrapper" role="search" aria-label="ResearchNow">
-          <div className="nypl-row">
-            <div className="nypl-column-full">
-              <h1 className="nypl-heading">ResearchNow</h1>
-              <div id="tagline">
-                Search the world&apos;s research collections and more for digital books you
-                can use right now.
+        <div className="nypl-full-width-wrapper">
+          <div className="nypl-page-header">
+            <Breadcrumbs query={this.props.searchQuery} pageType={pageType} />
+          </div>
+          <div role="search" aria-label="ResearchNow">
```

Comment:
I think you want to use `role="search"` with the form https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role.

Reply:
We should add role to the form as per best practice. Am asking for clarification from UX and Accessibility whether this landmark is appropriately placed or if it should be adjusted. Will wait for their answer and add as a separate PR.

Reply:
What was the consensus from the UX team?